noetic/distribution.yaml

```diff
@@ -3912,6 +3912,13 @@ repositories:
       url: https://github.com/facontidavide/plotjuggler_msgs.git
       version: ros1
     status: developed
+  plotjuggler_ros:
+    release:
```

Comment:
You had `doc` and `source` entries for `melodic`.  Did you intend to leave them out here?

Reply:
@facontidavide ping

Reply:
Running **bloom** again I am not prompted to add this information. Do you know how I can do that?

Reply:
You can do it manually if you wish.